cluster/saltbase/salt/kube-addons/kube-addons.sh

```diff
@@ -21,6 +21,33 @@ KUBECTL=${KUBECTL_BIN:-/usr/local/bin/kubectl}
 
 ADDON_CHECK_INTERVAL_SEC=${TEST_ADDON_CHECK_INTERVAL_SEC:-600}
 
+SYSTEM_NAMESPACE=kube-system
+
+function delete-if-present() {
```

Comment:
Yes, now I think it is dead code. If we remove it we will need to do another e2e run which means there is no hope of getting this merged today. We could leave it in and remove it post merge early next week.